pkgs/tools/networking/xxh/default.nix

```diff
@@ -1,19 +1,16 @@
 { lib
 , fetchFromGitHub
-, buildPythonApplication
-, pexpect
-, pyyaml
+, python3
 , openssh
 , nixosTests
-, pythonOlder
 }:
 
-buildPythonApplication rec{
+python3.pkgs.buildPythonApplication rec{
```

Comment:
Why not use `buildPythonPackage` or `toPythonModule` if that's what the error suggests? I don't think this is the correct way to package python applications such as this.

Reply:
but this is a application, not a library

@pasqui23 opinions?

Reply:
I've looked it up in the manual again and apparently this is indeed the intended way for packages that are pure applications (provide no python library).

Only for apps that also provide a library should you make them part of the `pythonPackages` set using `buildPythonPackage` and then `toPythonApplication` them in all-packages.